components/hero/Mission.tsx

```diff
@@ -0,0 +1,121 @@
+import { BitcoinIcon, EyeOffIcon } from "lucide-react"
+import Link from "next/link"
+
+export default function Mission() {
+    const features2 = [
+        {
+            badge: "EDUCATION",
+            name: "01 - Study",
+            description: "Learning Bitcoin is Hard we make it easy for you.",
+            href: "/bitcoin-core",
+            cta: "Check out topics",
+            icon: BitcoinIcon
+        },
+        {
+            badge: "CONTRIBUTION",
+            name: "02 - Contribute",
+            description:
+                "Put your knowledge to test. Explore vetted Free Open Source Software (FOSS) projects and find your way in bitcoin open source.",
+            href: "/good-first-issues",
+            cta: "Start Contribute",
+            icon: BitcoinIcon
+        },
+
+        {
+            badge: "TOOLS",
+            name: "03 - Build",
+            description:
+                "Explore a suite of tools crafted to support your learning, building, and contributions in the Bitcoin ecosystem.",
```

Comment:
So far, we've used "bitcoin" in lowercase and not capitalized. We should be consistent in this regard